config.go

```diff
@@ -51,54 +51,53 @@ type ConfigSection interface {
+	GithubPRCreatorOrg string                    `yaml:"github_pr_creator_org" bson:"github_pr_creator_org" json:"github_pr_creator_org"`
+	GithubOrgs         []string                  `yaml:"github_orgs" bson:"github_orgs" json:"github_orgs"`
+	HostInit           HostInitConfig            `yaml:"hostinit" bson:"hostinit" json:"hostinit" id:"hostinit"`
+	HostJasper         HostJasperConfig          `yaml:"host_jasper" bson:"host_jasper" json:"host_jasper" id:"host_jasper"`
+	Jira               JiraConfig                `yaml:"jira" bson:"jira" json:"jira" id:"jira"`
+	JIRANotifications  JIRANotificationsConfig   `yaml:"jira_notifications" json:"jira_notifications" bson:"jira_notifications" id:"jira_notifications"`
+	Keys               map[string]string         `yaml:"keys" bson:"keys" json:"keys"`
+	KeysNew            util.KeyValuePairSlice    `yaml:"keys_new" bson:"keys_new" json:"keys_new"`
+	LDAPRoleMap        LDAPRoleMap               `yaml:"ldap_role_map" bson:"ldap_role_map" json:"ldap_role_map"`
+	LoggerConfig       LoggerConfig              `yaml:"logger_config" bson:"logger_config" json:"logger_config" id:"logger_config"`
+	LogPath            string                    `yaml:"log_path" bson:"log_path" json:"log_path"`
+	NewRelic           NewRelicConfig            `yaml:"newrelic" bson:"newrelic" json:"newrelic" id:"newrelic"`
+	Notify             NotifyConfig              `yaml:"notify" bson:"notify" json:"notify" id:"notify"`
+	Plugins            PluginConfig              `yaml:"plugins" bson:"plugins" json:"plugins"`
+	PluginsNew         util.KeyValuePairSlice    `yaml:"plugins_new" bson:"plugins_new" json:"plugins_new"`
+	PprofPort          string                    `yaml:"pprof_port" bson:"pprof_port" json:"pprof_port"`
+	Providers          CloudProviders            `yaml:"providers" bson:"providers" json:"providers" id:"providers"`
+	RepoTracker        RepoTrackerConfig         `yaml:"repotracker" bson:"repotracker" json:"repotracker" id:"repotracker"`
+	Scheduler          SchedulerConfig           `yaml:"scheduler" bson:"scheduler" json:"scheduler" id:"scheduler"`
+	ServiceFlags       ServiceFlags              `bson:"service_flags" json:"service_flags" id:"service_flags" yaml:"service_flags"`
+	SSHKeyDirectory    string                    `yaml:"ssh_key_directory" bson:"ssh_key_directory" json:"ssh_key_directory"`
+	SSHKeyPairs        []SSHKeyPair              `yaml:"ssh_key_pairs" bson:"ssh_key_pairs" json:"ssh_key_pairs"`
+	Slack              SlackConfig               `yaml:"slack" bson:"slack" json:"slack" id:"slack"`
+	Splunk             send.SplunkConnectionInfo `yaml:"splunk" bson:"splunk" json:"splunk"`
+	Triggers           TriggerConfig             `yaml:"triggers" bson:"triggers" json:"triggers" id:"triggers"`
+	Ui                 UIConfig                  `yaml:"ui" bson:"ui" json:"ui" id:"ui"`
+	Spawnhost          SpawnhostConfig           `yaml:"spawnhost" bson:"spawnhost" json:"spawnhost" id:"spawnhost"`
```

Comment:
One note if you're creating a new section in the config to keep track of spawn host info, you should check what the defaults are currently and either pre-add them to the DB or add them back in from the UI immediately after deploying.

Reply:
Also a nit: Since "spawn host" is considered two words I think it's more correct for this to be SpawnHost

Reply:
😄 I've wondered about this.
done (spawnhost -> spawnHost)